.appveyor.yml

```diff
@@ -21,7 +21,7 @@ install:
   - "%PYTHON%\\Scripts\\pip.exe install --upgrade setuptools"
 
   # for now...  should somehow be managed via setup.py
-  - "%PYTHON%\\Scripts\\pip.exe install --upgrade msgpack-python cryptography requests tornado"
+  - "%PYTHON%\\Scripts\\pip.exe install --upgrade msgpack-python cryptography requests tornado sortedcontainers"
```

Comment:
Not blocking this PR, but I think `python setup.py install` is the command this comment is looking for.